### PR TITLE
71-Footer-padding-and-fontColor

### LINKED ISSUE #71: Footer - padding and fontColor

### PR DESCRIPTION
**Pull request by sergiobtos**
-Change Footer Padding
-Change font color using global css

### CODE REVIEW

File: src/styles/_variables.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 export const // Media Query Breakpoints
   $mobileBreakpoint = '578px',
   $tabletBreakpoint = '768px',
-  $lgDesktopBreakpoint = '1440px'
+  $lgDesktopBreakpoint = '1200px'
```
Comment (rmc3408) [inline src/styles/_variables.js]: i thought we decide 1280... but dont worry about this file. we neeed to rewrite.

**Comment by rmc3408** [comment]
Since you are doing cleaning... @sergiobtos 

can you remove the following file:

- remove .keep from non-empty folders.
- src/app/_document.tsx
- on Login and Signup, remove id for main HTML. And change the text for page (not this is a page -> to Login Page).
- on Header folder, change to style.ts not TSX.
- on Hero, hero.tsx to index.tsx and import/export to index.

- Logo is on `Assest` or `components`.... I rather on `Styles` folder.  like input , buttons, search bar, SVG , all together.
- Types folder, please rename those files activityType.ts for activity only . As well as the other files.

File: src/app/layout.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import { CssBaseline } from '@mui/material'
 import type { Metadata } from 'next'
 import { Roboto } from 'next/font/google'
-import Header from '../components/Header/Header'
-import { Footer } from '../components'
+import { Footer, Header} from '../components'
+import './global.css'
 
 //TODO for create fonts configuration variable.
 const roboto = Roboto({
```
Comment (rmc3408) [inline src/app/layout.tsx:9]: i guess it can remove this Roboto from line 8, right?

File: src/components/Footer/index.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 'use client'
-import { Logo } from '@/src/assets/svg'
+import Logo from '../Logo/Logo'
```
Comment (rmc3408) [inline src/components/Footer/index.tsx]: We will move all SVG to one folder. `src/styles` folder.

File: src/components/Footer/styles.ts [hunk 4]
```diff
@@ -34,14 +34,15 @@ export const Container = styled.div`
...
+  padding-left: 24px;
+  padding-right: 24px;
   text-align: center;
 
   ${m.tablet(css`
     display: flex;
-    justify-content: space-between;
     text-align: left;
     align-items: flex-end;
+    justify-content: space-between;
```
Comment (rmc3408) [inline src/components/Footer/styles.ts:45]: thanks , much better this way.

**Review by rmc3408 — APPROVED** [review]
Can you do a cleaning ? or want to create a ticket for that ?

**Comment by rmc3408** [comment]
Please, rename the name of the PR folllowing convention -> [issuesID]- what ticket say

File: src/components/Footer/index.tsx (hunk 3 above)
Comment (sergiobtos) [reply src/components/Footer/index.tsx:2]: @rmc3408 do you want move one asset like SVG to a folder named styles which mean how the component looks like.

File: src/app/layout.tsx (hunk 2 above)
Comment (sergiobtos) [reply src/app/layout.tsx:8]: that why I need to discuss with you, I do not know who did this way, but we to talk about global css.

File: src/styles/_variables.js (hunk 1 above)
Comment (sergiobtos) [reply src/styles/_variables.js]: done

**Pull request merged**